frontend/src/views/SE/Tasks.vue

```diff
@@ -55,9 +64,10 @@ import { formatTimestamp, getDayDifference } from "@/utility";
   components: { DataTable, SoundIssuesList }
 })
 export default class Tasks extends Vue {
-  tasks: ITasks[] = [];
+  tasks: any[] = [];
```

Comment:
Why removing the strong typing here?

Reply:
Yes, I was going to ask you about it but by mistake I forgot it to ask you about that I was getting some problem so I think to change any to make same with the user list page.

My problem was resolve without this change I will change it and push it again 

Don't worry

Reply:
Done